src/component/SelectMenu/Select.scss

```diff
@@ -0,0 +1,3 @@
+.AsyncSelect {
+  border-radius: 100px;
+}
```

Comment:
I don't see this being used anywhere. Am I missing something?

Reply:
yeah that should be removed, it was the old styling before mui button component